src/_pytest/compat.py

```diff
@@ -347,3 +348,9 @@ def funcargnames(self):
 
         warnings.warn(FUNCARGNAMES, stacklevel=2)
         return self.fixturenames
+
+
+if sys.version_info < (3, 5, 2):
```

Comment:
I don't think we need to test for coverage for `<3.5.2`, so it is fine to skip this from coverage:

```suggestion
if sys.version_info < (3, 5, 2):  # pragma: no cover
```